### ISSUE TITLE
Laravel 11 Support

### ISSUE THREAD

**Issue by larson-carter** [thedevdojo/voyager#5846]
### Laravel version

11

### PHP version

8.2

### Voyager version

1.7

### Database

PostgresQL

### Description

There is not laravel 10 or really laravel 11 support. It would be ideal if we could modernize and develop the application to be inclusive of this new framework. 

### Steps to reproduce

Attempt to install 

### Expected behavior

Install and work as it does on older versions

**Comment by schruptor** [comment]
@larson-carter Laravel 10 support is available. It's in the non-default branch of version 1.7.(https://github.com/thedevdojo/voyager/tree/1.7) (https://github.com/thedevdojo/voyager/releases/tag/v1.7)

But Laravel 11 support would be really great.

**Comment by alexandrbarabolia** [comment]
We need laravel 11 :+1:

**Comment by larson-carter** [comment]
@schruptor Yes, laravel 10 works great on that 1.7 branch (I think that it is stable enough for it to be on the default branch, just needs to be merged). However, there is no Laravel 11 support.

Maybe if I get time in a few weekends I will come back and make 11 support unless someone beats me to it.

**Comment by cotiga** [comment]
I think it will take a while (if it will ever be done...), because the loss of Doctrine DBAL seems like a gigantic undertaking for Voyager to overcome 😔

**Comment by larson-carter** [comment]
@cotiga I completely agree. 11 was a big jump over 10, heck really 9. 

I do think that there are a different items that make this upgrade quite a challenge. 

I just do not know of any alternatives that do the same functionality and are as modern/flexible.

**Comment by kozimjon-malikov** [comment]
We have to create adaptable version for all

**Comment by duran004** [comment]
I don't think there is a big jump from laravel 10 to 11, just some folders have been deleted, a simpler skeleton and now the project is started via bootstrap, that's all.

**Comment by larson-carter** [comment]
@duran004 Laravel 10 to 11 is quite a massive jump under the hood. It is not just folders being deleted or hidden.

Internal kernel functionality has been significantly modified.

**Comment by soinalastudio** [comment]
Has anyone reviewed/tested this commit for a support of Laravel 11 https://github.com/thedevdojo/voyager/commit/a4e3eef027ecd23fecad00e74fb6ac30ebd9cd63

**Comment by jonathanribas** [comment]
Hi, 
Laravel 10 bug fixes are over since 7th August 2024, PHP `8.2` Security fixes until February 2025.
Someone started something here: https://github.com/thedevdojo/voyager/pull/5801

**Comment by bobbyiliev** [comment]
> Has anyone reviewed/tested this commit for a support of Laravel 11 [a4e3eef](https://github.com/thedevdojo/voyager/commit/a4e3eef027ecd23fecad00e74fb6ac30ebd9cd63)

That 1.8 branch is still a work in progress. It is still not fully compatible with Laravel 11 though as the TCG\Voyager\Database\Types; still uses the deprecated Doctrine DBAL.

A lot of the BREAD functionality really depends on that Doctrine DBAL package which is no longer compatible with Laravel 11.

<img width="681" alt="image" src="https://github.com/user-attachments/assets/881535f7-ccaa-46e1-9f04-e42061145e05">

**Comment by bobbyiliev** [comment]
Quick update on the above, the 1.8 branch now works with Laravel 11.

This release includes some bug fixes and package updates to make it compatible with Laravel 11 with one exception:

> [!WARNING]
> As of Laravel 11 the Doctrine DBAL package has been removed from Laravel. This currently breaks the Voyager Database Manager. We are working on a fix for this, but in the meantime, you can just use the Laravel Database migrations to manage your database instead. The Voyager database manager might get deprecated in the next release.

<img width="715" alt="image" src="https://github.com/user-attachments/assets/496614b5-087c-40bd-93e4-6e4363c6b9a1">

**Comment by pckz** [comment]
Hey guys, thanks for your amazing job with this lib. Any news with the Laravel 11 support?
Bread is not working for users entity

**Comment by bobbyiliev** [comment]
> Hey guys, thanks for your amazing job with this lib. Any news with the Laravel 11 support? Bread is not working for users entity

Hey!

Laravel 11 should mostly work with: https://github.com/thedevdojo/voyager/releases/tag/v1.8.0

However as per the warning in the release:

> As of Laravel 11 the Doctrine DBAL package has been removed from Laravel. This currently breaks the Voyager Database Manager. We are working on a fix for this, but in the meantime, you can just use the Laravel Database migrations to manage your database instead. The Voyager database manager might get deprecated in the next release.
